msteams-platform/apps-in-teams-meetings/create-apps-for-teams-meetings.md

```diff
@@ -0,0 +1,278 @@
+```
+
+*See* the [Bot Framework API reference](/azure/bot-service/rest-api/bot-framework-rest-connector-api-reference?view=azure-bot-service-4.0&preserve-view=true).
+
+<!-- markdownlint-disable MD025 -->
+
+# [C # /.NET](#tab/dotnet)
+
+```csharp
+string meetingId = "meetingid?";
+string participantId = "participantidhere";
+var connectorClient = turnContext.TurnState.Get<IConnectorClient>();
+var creds = connectorClient.Credentials as AppCredentials;
+var bearerToken = await creds.GetTokenAsync().ConfigureAwait(false);
+var request = new HttpRequestMessage();
+request.Headers.Authorization = new AuthenticationHeaderValue("Bearer", bearerToken);
+request.Method = new HttpMethod("GET");
+request.RequestUri = new System.Uri(Path.Combine(connectorClient.BaseUri.OriginalString, $"/meetings/{meetingId}/participants/{participantId}"));
+HttpResponseMessage response = await (connectorClient as ServiceClient<ConnectorClient>).HttpClient.SendAsync(request, cancellationToken).ConfigureAwait(false);
+if (response.StatusCode == System.Net.HttpStatusCode.OK)
+{
+    var content = await response.Content.ReadAsStringAsync().ConfigureAwait(false);
+    var theObject = Rest.Serialization.SafeJsonConvert.DeserializeObject<WhateverObjectIsReturned>(content, connectorClient.DeserializationSettings);
+```
+
+---
+<!-- markdownlint-disable MD001 -->
+
+#### Query parameters
+
+**meetingId**. The meeting identifier is required.  
+**participantId**. The participant identifier is required.  
+**tenantId**. [Tenant id](/onedrive/find-your-office-365-tenant-id) of the participant. Required for tenant user.
+
+#### Response Payload
+<!-- markdownlint-disable MD036 -->
+
+**Example 1**
+
+```json
+{
+    "meetingRole":"Presenter",
+    "conversation":
+        {
+            "isGroup": true,
+            "id": "19:meeting_NDQxMzg1YjUtMGIzNC00Yjc1LWFmYWYtYzk1MGY2MTMwNjE0@thread.v2"
+        }
+}
+```
+
+**meetingRole** can be *Organizer*, *Presenter*, or *Attendee*.
+
+**Example 2**
+
+```json
+{
+    "meetingRole": "Attendee",
+    "conversation": {
+        "isGroup": true,
+        "id": "19:meeting_OWIyYWVhZWMtM2ExMi00ZTc2LTg0OGEtYWNhMTM4MmZlZTNj@thread.v2" }
+}
+```
+
+#### Response Codes
+
+**403**: the app is not allowed to get participant information. This will be the most common error response and is triggered when the app is not installed in the meeting such as when the app is disabled by tenant admin or blocked during live site mitigation.
+**200**: participant information successfully retrieved  
+**401**: invalid token  
+**404**: the meeting doesn't exist or participant can’t be found.
+
+<!-- markdownlint-disable MD024 -->
+### NotificationSignal API
+
+#### Request
+
+```http
+POST /v3/conversations/{conversationId}/activities
+```
+
+#### Query parameters
+
+**conversationId**: The conversation identifier. Required
+
+#### Request Payload
+
+# [JSON](#tab/json)
+
+```json
+{
+      "type": "message",
+      "timestamp": "2017-04-24T21:46:00.9663655Z",
```

Comment:
```
      "timestamp": "2017-04-24T21:46:00.9663655Z",
      "localTimestamp": "2017-04-24T14:46:00.9663655-07:00",
      "serviceUrl": "https://callback.com",
      "channelId": "msteams",
      "from": {
        "id": "28:e4fda94a-4b80-40eb-9bf0-6314491bc793",
        "name": "The bot"
      },
      "conversation": {
        "id": "a:1pL6i0oY3C0K8oAj8"
      },
      "recipient": {
        "id": "29:1rsVJmSSFMScF0YFyCXpvNWlo",
        "name": "User"
      },
```
Let's remove these lines, and the `replyToId`, as they are not needed when sending an activity.